staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1600,12 +1600,26 @@ type MigrationConfiguration struct {
 // DeveloperConfiguration holds developer options
 // +k8s:openapi-gen=true
 type DeveloperConfiguration struct {
-	FeatureGates           []string          `json:"featureGates,omitempty"`
-	LessPVCSpaceToleration int               `json:"pvcTolerateLessSpaceUpToPercent,omitempty"`
-	MemoryOvercommit       int               `json:"memoryOvercommit,omitempty"`
-	NodeSelectors          map[string]string `json:"nodeSelectors,omitempty"`
-	UseEmulation           bool              `json:"useEmulation,omitempty"`
-	CPUAllocationRatio     int               `json:"cpuAllocationRatio,omitempty"`
+	FeatureGates           []string            `json:"featureGates,omitempty"`
+	LessPVCSpaceToleration int                 `json:"pvcTolerateLessSpaceUpToPercent,omitempty"`
+	MemoryOvercommit       int                 `json:"memoryOvercommit,omitempty"`
+	NodeSelectors          map[string]string   `json:"nodeSelectors,omitempty"`
+	UseEmulation           bool                `json:"useEmulation,omitempty"`
+	CPUAllocationRatio     int                 `json:"cpuAllocationRatio,omitempty"`
+	DebugConfiguration     *DebugConfiguration `json:"debugConfiguration,omitempty"`
+}
+
+// DebugConfiguration sets various components to a debug mode
+// +k8s:openapi-gen=true
+type DebugConfiguration struct {
+	DebugAllVirtComponents bool `json:"debugAllVirtComponents,omitempty"`
+	DebugVirtApi           bool `json:"debugVirtApi,omitempty"`
+	DebugVirtControllers   bool `json:"debugVirtControllers,omitempty"`
+	DebugVirtLaunchers     bool `json:"debugVirtLaunchers,omitempty"`
```

Comment:
This probably only debugs the go binary but not libvirt/qemu?

Reply:
Yep. I wasn't sure how to go about this. I didn't want to introduce a new tunable. I've kept the enablement of libvirt/qemu debug logs as la abel.
If we are switching to integers for verbosity, perhaps `6` may enable libvirt/qemu debug logs.